examples/x11ex.nim

```diff
@@ -42,7 +45,7 @@ proc create_window =
                                      PointerMotionMask or ExposureMask)
   discard XMapWindow(display, win)
 
-  wmDeleteMessage = XInternAtom(display, "WM_DELETE_WINDOW", false.TBool)
+  wmDeleteMessage = XInternAtom(display, "WM_DELETE_WINDOW", false.Bool)
```

Comment:
This must become ``X11Bool`` or similar, too subtle otherwise.

Reply:
@Araq How about XBool? To keep it the same length as before